### PR TITLE
*: cut 1.5.0

### PR DESCRIPTION
**Pull request by brancz**
Big release, let me know if I missed anything, tagged anything incorrectly or want the wording to be different. As always I tried to keep the changelog as user relevant as possible.

@brian-brazil @beorn7 @juliusv @fabxc @alexsomesan

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+* [CHANGE] Use lexicographic order to sort alerts by name
+* [FEATURE] Add Joyent Triton discovery
+* [FEATURE] Add scrape targets and alertmanager targets API
+* [FEATURE] Add various persistence related metrics
+* [FEATURE] Add various query engine related metrics
+* [FEATURE] Add ability to limit scrape samples, and related metrics
+* [FEATURE] Add labeldrop and labelkeep relabelling actions
+* [FEATURE] Display current working directory on status-page
+* [ENHANCEMENT] Allow checkpoints and maintenance to happen concurrently
+* [ENHANCEMENT] Strictly use ServiceAccount for in cluster configuration on Kubernetes
```
Comment (brian-brazil) [inline CHANGELOG.md]: There were also various performance improvements

Comment (brancz) [inline CHANGELOG.md]: Yes I wasn't sure how to describe them. Do you want me to just say "Various performance improvements" ?

Comment (brancz) [inline CHANGELOG.md]: Added

```
* [ENHANCEMENT] Various performance improvements
```

**Comment by alexsomesan** [comment]
lgtm.
my change is accurately phrased.

File: CHANGELOG.md (hunk 1 above)
Comment (brian-brazil) [inline CHANGELOG.md]: Yeah, various and memory improvements.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+* [CHANGE] Use lexicographic order to sort alerts by name
+* [FEATURE] Add Joyent Triton discovery
+* [FEATURE] Add scrape targets and alertmanager targets API
+* [FEATURE] Add various persistence related metrics
+* [FEATURE] Add various query engine related metrics
+* [FEATURE] Add ability to limit scrape samples, and related metrics
+* [FEATURE] Add labeldrop and labelkeep relabelling actions
+* [FEATURE] Display current working directory on status-page
+* [ENHANCEMENT] Allow checkpoints and maintenance to happen concurrently
```
Comment (beorn7) [inline CHANGELOG.md]: According to the result of the lengthy discussion recently, I'd say this is way to internal to mention it here. Users won't even recognize what "maintenance" is. I would just leave it out. It's kind of covered by the "various performance improvements".

Comment (brancz) [inline CHANGELOG.md]: Done.

**Comment by beorn7** [comment]
BTW: How confident does everybody feel about the changes? Should we cut an RC for this or go straight into the release?

**Comment by beorn7** [comment]
@juliusv -mode: We have inconsistencies about trailing periods. I'd consistently use a period at the end of each entry.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+* [CHANGE] Use lexicographic order to sort alerts by name
+* [FEATURE] Add Joyent Triton discovery
+* [FEATURE] Add scrape targets and alertmanager targets API
+* [FEATURE] Add various persistence related metrics
+* [FEATURE] Add various query engine related metrics
+* [FEATURE] Add ability to limit scrape samples, and related metrics
+* [FEATURE] Add labeldrop and labelkeep relabelling actions
+* [FEATURE] Display current working directory on status-page
+* [ENHANCEMENT] Strictly use ServiceAccount for in cluster configuration on Kubernetes
+* [ENHANCEMENT] Various performance improvements
```
Comment (beorn7) [inline CHANGELOG.md]: How about "Various performance and memory-management improvements."?

**Comment by brancz** [comment]
Periods and comment fixed. I think we should go straight for the release.

**Comment by beorn7** [comment]
:+1: 

Instructions are https://github.com/prometheus/prometheus/wiki/HOWTO-cut-a-new-release (just in case you aren't looking at those anyway).

**Comment by brancz** [comment]
Didn't see the document but was following that pretty much by looking at previous releases. Just one question .. I'm guessing the `stable` branch is force pushed occasionally then? I can imagine it might have conflicts often.

**Comment by beorn7** [comment]
On 23 January 2017 at 13:57, Frederic Branczyk <notifications@github.com>
wrote:

> I'm guessing the stable branch is force pushed occasionally then? I can
> imagine it might have conflicts often.
>
Yes. I can imagine using a stable _branch_ is not quite the right thing WRT
Git semantics. It should probably be a tag or something... perhaps it is
just legacy. To be discussed...


-- 
Björn Rabenstein, Engineer
http://soundcloud.com/brabenstein

SoundCloud Ltd. | Rheinsberger Str. 76/77, 10115 Berlin, Germany
Managing Director: Alexander Ljung | Incorporated in England & Wales with
Company No. 6343600 | Local Branch Office | AG Charlottenburg  | HRB 110657B

**Comment by brancz** [comment]
Alright then, just wanted to make sure. Maybe @brian-brazil can confirm the changes then we can move forward with this release.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**